src/Tizen.NUI/src/public/Common/Vector4.cs

```diff
@@ -20,6 +20,32 @@
 
 namespace Tizen.NUI
 {
+    internal class DaliVector4 : Disposable
+    {
+        public DaliVector4() : base(Interop.Vector4.NewVector4(), true)
```

Comment:
true is passed as a parameter.
maybe false is better here? 
if false, the Disposable above can be removed.
if false, a static variable need be added at csharp-binder side.